src/components/context/DataProvider.tsx

```diff
@@ -257,9 +256,7 @@ const DataProvider = ({ children }: Props): JSX.Element => {
     ],
   );
 
-  return (
-    <DataContext.Provider value={value}> {children} </DataContext.Provider>
-  );
+  return <DataContext.Provider value={value}> {children}</DataContext.Provider>;
```

Comment:
```suggestion
  return <DataContext.Provider value={value}> {children} </DataContext.Provider>;
```

Reply:
I don't know why it is formatted like that ...